ducktape/command_line/main.py

```diff
@@ -35,9 +39,25 @@ def parse_args():
                         help='one or more space-delimited strings indicating where to search for tests')
     parser.add_argument("--collect-only", action="store_true", help="display collected tests, but do not run")
     parser.add_argument("--debug", action="store_true", help="pipe more verbose test output to stdout")
+    parser.add_argument("--config-file", action="store", default=ConsoleConfig.PROJECT_CONFIG_FILE,
+                        help="path to project-specific configuration file")
+    parser.add_argument("--cluster", action="store", default=ConsoleConfig.CLUSTER_TYPE,
+                        help="cluster class to use to allocate nodes for tests")
     parser.add_argument("--exit-first", action="store_true", help="exit after first failure")
 
-    args = parser.parse_args()
+    project_configs = []
+    project_config_file = ConsoleConfig.PROJECT_CONFIG_FILE
+    if os.path.exists(project_config_file):
+        project_configs = list(itertools.chain(*[line.split() for line in open(project_config_file).readlines()]))
+
+    user_configs = []
+    user_config_file = os.path.expanduser(ConsoleConfig.USER_CONFIG_FILE)
+    if os.path.exists(user_config_file):
+        user_configs = list(itertools.chain(*[line.split() for line in open(user_config_file).readlines()]))
+
+    # The order of concatenation ensures overrides are handled properly -- for action=store arguments, the later options
+    # will override earlier instances
+    args = parser.parse_args(project_configs + user_configs + sys.argv[1:])
```

Comment:
How about with --debug? I'm skeptical of adding that output to every run since it adds noise.